common/src/main/java/com/microsoft/identity/common/internal/authorities/Authority.java

```diff
@@ -279,13 +279,16 @@ protected boolean getKnownToDeveloper() {
 
     private static void performCloudDiscovery() throws IOException {
         final String methodName = ":performCloudDiscovery";
-        Logger.verbose(
+        Logger.info(
                 TAG + methodName,
                 "Performing cloud discovery..."
         );
         synchronized (sLock) {
+            Logger.info(TAG + methodName, "Acquired lock.");
```

Comment:
I guess this can be `verbose`

Reply:
CP isn't logging anything below `info` -- in this case, I want to know if the lock is taking a long time to be acquired